wikidata_position_history.gemspec

```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "wikidata_position_history/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "wikidata_position_history"
+  spec.version       = WikidataPositionHistory::VERSION
+  spec.authors       = ["Tony Bowden"]
+  spec.email         = ["team@everypolitician.org"]
+
+  spec.summary       = %q{Generates a wikitext history of a holders of a position in Wikidata}
+  spec.homepage      = "https://github.com/everypolitician/wikidata-position-history/"
+  spec.license       = "MIT"
+
+  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the 'allowed_push_host'
+  # to allow pushing to a single host or delete this section to allow pushing to any host.
+  if spec.respond_to?(:metadata)
+    spec.metadata["allowed_push_host"] = "TODO: Set to 'http://mygemserver.com'"
+  else
+    raise "RubyGems 2.0 or newer is required to protect against " \
+      "public gem pushes."
+  end
+
+  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
+    f.match(%r{^(test|spec|features)/})
+  end
+  spec.bindir        = "exe"
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"]
+
+  spec.add_runtime_dependency 'mediawiki-page-replaceable_content', '0.1.2'
+  spec.add_runtime_dependency 'rest-client', '~> 2.0'
+
+  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "minitest", "~> 5.0"
+  spec.add_development_dependency 'pry', '~> 0.10'
+  spec.add_development_dependency 'webmock', '~> 3.0.0'
```

Comment:
The mismatched quote style here is a little jarring. I think it's worth running everything through `rubocop -a` — it's OK to do that as a commit of its own at the end if you don't want to rewrite each existing commit.

Reply:
I've fixed the gemspec by rewriting the history, but the other changes it suggests I've added in a new commit.